.github/workflows/build.yaml

```diff
@@ -54,6 +54,12 @@ jobs:
       - name: Build (sev_snp)
         run: cargo rustc --locked --bin cloud-hypervisor --no-default-features --features "sev_snp"  -- -D warnings -D clippy::undocumented_unsafe_blocks
 
+      - name: Build (mshv + igvm)
+        run: cargo rustc --locked --bin cloud-hypervisor --no-default-features --features "mshv,igvm"  -- -D warnings -D clippy::undocumented_unsafe_blocks
+
```

Comment:
This use case is for development purpose if anyone want to boot the guest using igvm without sev-snp.

Reply:
It would be good to have an overview of relationship among these new features (igvm, sev_snp) and also existing features (say mshv, kvm, tdx) if related, plus the use-cases behind them as well as if they will be supported via this set incoming PRs. 

In general, we should enforce feature dependencies explicitly from cargo.toml based on supported use-cases.

Reply:
I can create a separate PR for documenting mshv, igvm and sev_snp features details and relations.

Reply:
This would be great. It will be very helpful for reviewers and users who do not know much about igvm/sev_snp/mshv. Thanks.